modules/restful_token_auth/templates/user-key.tpl.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Default theme implementation to present a user's API token plus an
+ * example with next steps.
+ *
+ * Available variables:
+ * - $token: The API token.
+ *
+ * @ingroup themeable
+ */
+?>
+<p>Your authentication token is: <code><?php print $token; ?></code></p>
+
+<p>Discover the API by running the following command in a terminal and replacing
+    [your-token] by the above string:</p>
+
+<p>
+    <code>
+        curl -H 'access-token:<?php print $token; ?>' <?php print $GLOBALS['base_url'] ?>/api?all=true
```

Comment:
Maybe it's better to have a preprocess function that uses `url(…)` to set up a variable. Then we only need to print the variable here. The current code will fail if the site is installed in a subdir, or there is a language path prefix, etc.